test/test_jit_fuser_randomized.py

```diff
@@ -0,0 +1,41 @@
+# Owner(s): ["NNC"]
+
+import torch
+import torch.nn.functional as F
+import os
+
+# these needs to be set before `common_utils`
+# infers `GRAPH_EXECUTOR`.
+# this file **requires** these settings
+# and setting them after `GRAPH_EXECUTOR` is
+# inferred erroneously runs or skips
+# some tests
+torch._C._jit_set_profiling_executor(True)
+torch._C._jit_set_profiling_mode(True)
+
+
+from torch.testing._internal.common_utils import run_tests, TestCase
+
+
+class TestRandomized(TestCase):
+    def test_relu(self):
+        def fn_test_relu(x, y):
+            return F.relu(x + 0.5 * y)
+
+        # Set the seed to 1. This tests the codepath through random
+        # transformation.
+        os.environ["PYTORCH_TENSOREXPR_RANDOM_TRANSFORM_SEED"] = "1"
```

Comment:
Does it not affect other tests running in CI? I.e. can this cause random transforms to be enabled in other tests that happen to run after this?

Reply:
Oh I did not think of that possibility. Let me try and see if there is any better way.

Reply:
Added `setUp` and `tearDown` functions, so it should alter the environment only during the specific test.